cmd/caboose/main.go

```diff
@@ -38,7 +38,7 @@ func main1() int {
 			}
 			out := args.Get(1)
 
-			oe, _ := url.Parse("https://orchestrator.strn.pl/nodes/nearby")
+			oe, _ := url.Parse("https://orchestrator.strn.pl/nodes/nearby?count=100")
```

Comment:
This parameter should be moved inside of caboose asa a `const`, appended to the endpoint.
You don't want to ask bifrost-gateway to change URL every time you want to adjust strategy for refreshing L1s :)